rust/signer/src/lib.rs

```diff
@@ -223,6 +223,10 @@ fn history_seed_name_was_shown(seed_name: &str, dbname: &str) -> anyhow::Result<
         .map_err(|e| format!("{}", e))
 }
 
+fn export_key_info(dbname: &str) -> anyhow::Result<MKeysInfoExport, String> {
```

Comment:
Please expose existing function to get bitmap qr code from vec<u8> as well. In addition to current request. It will be easier to use it for now rather than inventing logic on android side. 
There is no native solution and all decent libraries I see areworking with string exclusively. 
I can request it 1 by 1 in parallel or as an array in 1 shot - doesn't matter.

Reply:
Done